CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## 2.24.0-rc.0 / 2020-12-30
+
+* [FEATURE] Add TLS and basic authentication to HTTP endpoints. #8316
+* [FEATURE] promtool: Add `check web-config` command to check web config files. #8319
+* [FEATURE] promtool: Add `tsdb create-blocks-from openmetrics` command to backfill metrics data from an OpenMetrics file. #8084
+* [ENHANCEMENT] HTTP API: Fast-fail queries with only empty matchers. #8288
+* [ENHANCEMENT] HTTP API: Support matchers for labels API. #8301
+* [ENHANCEMENT] promtool: Improve checking of URLs passed on the command line. #7956
+* [ENHANCEMENT] SD: Expose IPv6 as a label in EC2 SD. #7086
+* [ENHANCEMENT] SD: Reuse EC2 client, reducing frequency of requesting credentials. #8311
+* [ENHANCEMENT] TSDB: Add logging when compaction takes more than the block time range. #8151
+* [ENHANCEMENT] TSDB: Reduced log level for overlapping blocks from warn to info. 8313
```

Comment:
We don't generally list logging changes in the changelog, they're too minor.

Reply:
This might have annoyed or confused people quite a bit, so I thought in this case, it's worth mentioning.

Also, some might actually have scenarios where they _want_ to see those messages, and then they should know that they have to change the level. (I was tempted to list this as `[CHANGE]` but then decided that it would draw too much attention. But I definitely don't want to not mention this at all.)

Reply:
That still doesn't meet notability in my mind. If you were seeing these in a standard Prometheus setup then compaction was broken, so this is not important to the typical user as they've far bigger problems and the message is still there one way or the other. In fact this change was for the sake of Cortex/Thanos, not Prometheus.

Adding, removing, or changing log messages don't belong in the changelog unless there's some major impact to users somehow. Otherwise we'd be spamming the changelog with every minor log line tweak. Logging changes are never a CHANGE.

Reply:
I agree with Brian, even more since this have been requested by Cortex users, not by Prometheus users.

Reply:
I still disagree, but once more, this is not a battle I'll pick.